mozilla_django_oidc/views.py

```diff
@@ -111,12 +110,8 @@ def get_next_url(request, redirect_field_name):
             'require_https': request.is_secure()
         }
 
-        # NOTE(robhudson): Django 2.1 changes `host` to `allowed_hosts`.
         host = request.get_host()
-        if django.VERSION >= (2, 1):
```

Comment:
Shouldn't this stay?

Reply:
No the django code internally handles all versions > 1.8